{"name":"Sign Up Website","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [true] but found [false]","trace":"java.lang.AssertionError: expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertTrue(Assert.java:44)\n\tat org.testng.Assert.assertTrue(Assert.java:54)\n\tat tests.ProductStore.SignUpTest.InvalidUserNameSignUp(SignUpTest.java:36)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","description":"Testcase description : Verify Sign Up","steps":[],"attachments":[{"name":"Page screenshot","source":"f07124fc-0f09-40ce-9744-0956d22effef-attachment","type":"image/png"},{"name":"InvalidUserNameSignUp failed and screenshot taken!","source":"ae367af2-5d78-4022-b53f-e07d64d0dcfe-attachment","type":"text/plain"}],"parameters":[],"start":1606989684501,"stop":1606989686166,"uuid":"5f24cafc-7074-47da-996b-3796f49fe2f9","historyId":"94480a195dd7240abe169cbeaec61902","fullName":"tests.ProductStore.SignUpTest.InvalidUserNameSignUp","labels":[{"name":"package","value":"tests.ProductStore.SignUpTest"},{"name":"testClass","value":"tests.ProductStore.SignUpTest"},{"name":"testMethod","value":"InvalidUserNameSignUp"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Selenium-framwork"},{"name":"subSuite","value":"tests.ProductStore.SignUpTest"},{"name":"host","value":"BAEMINs-MacBook-Pro.local"},{"name":"thread","value":"9898@BAEMINs-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"blocker"}],"links":[]}